src/ui/components/Button/index.js

```diff
@@ -23,7 +26,11 @@ export default class Button extends React.Component {
       to,
       ...rest
     } = this.props;
-    const props = { className: classNames('Button', className), ...rest };
+    const props = { ...rest };
+
+    const setClassName = (...classConfig) => {
+      return classNames('Button', className, ...classConfig);
```

Comment:
Nit but this will output just the class `disabled`, right? I'd think we'd want `Button--disabled`... I think this would work with `classConfig.map((name) => `Button--${name}`)`

Reply:
Yes, I intended to use the general purpose [`.disabled`](https://github.com/mozilla/addons-frontend/blob/master/src/core/css/inc/mixins.scss#L261) style. Were you thinking of `InstallButton--disabled`?